### PR TITLE
fix: Chain Integration doc refactor

### PR DESCRIPTION
**Pull request by ffe9f8**
Bringing in @canhtrinh's changes from [PR 847](https://github.com/axelarnetwork/axelar-docs/pull/847).

### CODE REVIEW

File: src/pages/dev/amplifier/chain-integration.mdx [hunk 1]
```diff
@@ -8,13 +8,13 @@ The Axelar Virtual Machine (AVM) and Amplifier are currently under active develo
...
+- You should know how to deploy and interact with a CosmWasm contract. Refer to the [Osmosis CosmWasm testnet deployment tutorial](https://docs.osmosis.zone/cosmwasm/testnet/cosmwasm-deployment/) if you’d like a refresher — just replace `osmosisid` with `axelard` and use the `amd64` version if building on Linux and `arm64` version of the [contract optimizer](https://github.com/CosmWasm/optimizer) if you are working on a [Mac with Apple silicon](https://support.apple.com/en-us/116943). Run `--gas auto --gas-adjustment 1.5` to provide your transactions with sufficient gas.
 
 ## Devnet information
 
-- Devnet RPC: `http://devnet-amplifier.axelar.dev:26657`
-- Devnet LCD (REST API/JSON RPC): `http://devnet-amplifier.axelar.dev:1317`
-- Devnet gRPC: `devnet-amplifier.axelar.dev:9090`
+- Devnet RPC: [http://devnet-amplifier.axelar.dev:26657](http://devnet-amplifier.axelar.dev:26657)
```
Comment (StephenFluin) [inline src/pages/dev/amplifier/chain-integration.mdx:15]: This isn't a URL you can visit I think, so does linking help?

File: src/pages/dev/amplifier/chain-integration.mdx [hunk 2]
```diff
@@ -8,13 +8,13 @@ The Axelar Virtual Machine (AVM) and Amplifier are currently under active develo
...
+- Devnet LCD (REST API/JSON RPC): [http://devnet-amplifier.axelar.dev:1317](http://devnet-amplifier.axelar.dev:1317)
+- Devnet gRPC: [devnet-amplifier.axelar.dev:9090](devnet-amplifier.axelar.dev:9090)
```
Comment (StephenFluin) [inline src/pages/dev/amplifier/chain-integration.mdx:17]: Ditto for the others

**Review by StephenFluin — APPROVED** [review]

**Comment by StephenFluin** [comment]
@ffe9f8 approved but there are conflicts.

File: src/pages/dev/amplifier/chain-integration.mdx [hunk 3]
```diff
@@ -41,55 +41,74 @@ See the [Become an Amplifier Verifier (Worker)](/validator/amplifier/verifier-on
...
-1. Build all of the deployed contracts, such as the router and service registry, from [`devnet-amplifier.json`](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/axelar-chains-config/info/devnet-amplifier.json) using the [Cosmwasm `rust-optimizer`](https://github.com/CosmWasm/optimizer). From the root of the repo, run:
+1. Build and deploy the contracts. The primary contracts are the `gateway`, `voting verifier`, and `multisig prover` contracts. You can either:
 
-    ```bash
-    docker run --rm -v "$(pwd)":/code \
-    --mount type=volume,source="$(basename "$(pwd)")_cache",target=/target \
-    --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-    cosmwasm/optimizer:0.15.0
-    ```
+    * Use the existing deployments for all three contracts, making node of the existing deployment's `code id`:
```
Comment (canhtrinh) [inline src/pages/dev/amplifier/chain-integration.mdx:46]:
```suggestion
    * Use the existing deployments for all three contracts, making note of the existing deployment's `code id`:
```

**Review by StephenFluin — APPROVED** [review]

**Pull request merged**